promql/promqltest/testdata/aggregators.test

```diff
@@ -235,21 +235,25 @@ eval instant at 0m max(http_requests)
   {} 4
```

Comment:
Let's add an `expect no_info` here to emphasize the difference to the tests below.